pom.xml

```diff
@@ -52,7 +52,7 @@
         <exclude.quarkus.cli.tests>**/QuarkusCli*IT.java</exclude.quarkus.cli.tests>
         <exclude.quarkus.devmode.tests>no</exclude.quarkus.devmode.tests>
         <!-- Docker images used by both surefire and failsafe plugin -->
-        <postgresql.latest.image>docker.io/library/postgres:16.1</postgresql.latest.image>
+        <postgresql.latest.image>docker.io/library/postgres:16.3</postgresql.latest.image>
```

Comment:
Afaik should be better to use just `docker.io/library/postgres:16` without the minor version as it should always download the latest

Reply:
Good point, changed it.